include/openssl/ssl.h

```diff
@@ -1899,6 +1899,15 @@ int DTLSv1_listen(SSL *s, BIO_ADDR *client);
 # ifndef OPENSSL_NO_CT
 
 /*
+ * A callback for verifying that the received SCTs are sufficient.
+ * Expected to return 1 if they are sufficient, otherwise 0.
+ * May return a negative integer if an error occurs.
```

Comment:
Hm. I don't know. As with everything we don't really have a convention. But I think if you're loose you should be loose in both directions. What happens now if you return 2? Maybe document that it should return 1 for success, 0 for error, and in code just compare `== 1` and error on anything else?

I was going to suggest to follow https://www.openssl.org/docs/manmaster/ssl/SSL_CTX_set_alpn_select_cb.html but  sadly `SSL_TLSEXT_ERR_OK` is 0 and I don't like reversing the usual return value convention so maybe we shouldn't let it spread.